packages/terraform-modules-public/aws/testnet/modules/startup-scripts/run-attestation-service.sh

```diff
@@ -9,13 +9,23 @@ mkdir $NODE_DIRECTORY
 cd $NODE_DIRECTORY
 
 docker run -v $PWD:/root/.celo --rm $CELO_IMAGE init /celo/genesis.json
-export BOOTNODE_ENODES=`docker run --rm --entrypoint cat $CELO_IMAGE /celo/bootnodes`
+export BOOTNODE_ENODES="$(docker run --rm --entrypoint cat $CELO_IMAGE /celo/bootnodes)"
 
 export CELO_ATTESTATION_SIGNER_ADDRESS=${attestation_signer_address}
 echo -n '${attestation_signer_private_key_password}' > .password
 echo -n '${attestation_signer_private_key_file_contents}' > keystore/${attestation_signer_private_key_filename}
 
-docker run -d --name celo-attestations --restart always -p 127.0.0.1:8545:8545 -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --nousb --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin --unlock $CELO_ATTESTATION_SIGNER_ADDRESS --password /root/.celo/.password --bootnodes $BOOTNODE_ENODES
+ATTESTATION_NODE_CLOUDWATCH_LOG_GROUP_NAME=${cloudwatch_attestation_node_log_group_name}
+ATTESTATION_NODE_CLOUDWATCH_LOG_STREAM_NAME=${cloudwatch_attestation_node_log_stream_name}
+
+if [[ -z $ATTESTATION_NODE_CLOUDWATCH_LOG_GROUP_NAME || -z $ATTESTATION_NODE_CLOUDWATCH_LOG_STREAM_NAME ]]; then
+  DOCKER_LOGGING_PARAMS=''
+else
+  DOCKER_LOGGING_PARAMS="--log-driver=awslogs --log-opt awslogs-group=$ATTESTATION_NODE_CLOUDWATCH_LOG_GROUP_NAME --log-opt awslogs-stream=$ATTESTATION_NODE_CLOUDWATCH_LOG_STREAM_NAME"
+fi
+
+docker run -d --name celo-attestations $DOCKER_LOGGING_PARAMS --restart always -p 127.0.0.1:8545:8545 -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --nousb --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin --unlock 0x$CELO_ATTESTATION_SIGNER_ADDRESS --password /root/.celo/.password --bootnodes $BOOTNODE_ENODES --allow-insecure-unlock
```

Comment:
Sometimes verbosity `3` could be too verbose, but it could help with debugging. What is your opinion having this as a variable? For having only a few instances I think 3 is ok, but when you deploy more or with testnets sometimes makes sense to have in 1 to save some dollars on monitoring.

Reply:
I like the idea of making this a parameter. Will consider this in a future PR.

Re: dollars - in my experience the monitoring $$ is dwarfed by compute costs, so probably not a huge cost savings here.

Reply:
:+1: Haha, in my experience I saw several times the monitoring bill was the highest service used :), but probably caused by too verbose services.